test/integration/update_int_test.go

```diff
@@ -276,14 +276,22 @@ func (suite *UpdateIntegrationTestSuite) TestUpdate() {
 			err := fileutils.Mkdir(fakeHome)
 			suite.Require().NoError(err)
 
+			before := fileutils.ListDir(ts.Dirs.Config, false)
```

Comment:
Just suggest to use this instead: https://github.com/ActiveState/cli/pull/1465/commits/7d26798eab86188ecebc4cdb1d40f5a069312f70